pages/science/reactions.rst

```diff
@@ -285,6 +285,47 @@ Chebyshev reactions can be defined in the CTI format using the
 :cti:class:`chebyshev_reaction` entry, or in the YAML format using the
 :ref:`Chebyshev <sec-yaml-Chebyshev>` reaction ``type``.
 
+.. _sec-Blowers-Masel:
+
+Blowers-Masel Reactions
+-----------------------
+
+Blowers-Masel rate expression represents the approximation porposed by Blowers
+and Masel [#BlowersMasel2000]_ to automatically scale activation energy
+as we change species' enthalpies. The activation energy estimation can be written
+as:
+
+.. math::
+
+   E_a = 0 \;\text{if } \Delta H \leq -4 E_a^0
+
+   E_a = \Delta H \;\text{if } \Delta H \geq 4 E_a^0
+
+   E_a =  \frac{\left( w + \frac{\Delta H }{2} \right)  (V_P - 2 w + \Delta H) ^2}
+               {V_P^2 - 4 w^2 + \Delta H^2} \;\text{Otherwise}
+
+where
+
+.. math::
+
+   V_P = 2 w \frac{w + E_a^0}{w - E_a^0},
+
+:math:`w` is the average of the bond dissociation energy of the bond breaking and that being formed,
+:math:`E_a^0` is the intrinsic activation energy, and :math:`\Delta H` is the enthalpy change of the reaction.
+Note that the expression is very insensitive to :math:`w` as long as :math:`w \ge 2 E_a^0`, so we can use
+an arbitrary high value of :math:`w = 1000 kJ/mol`.
+
+After :math:`E_a` is evaluated, the reaction rate can be calculated as modified Arrhenius expression
+
+.. math::
+
+   k_f = A T^b e^{-E_a / RT}.
+
+Blowers Masel reaction can be defined in the YAML format using the
+:ref:`Blowers-Masel <sec-yaml-Blowers-Masel>` reaction ``type``.
```

Comment:
For now, you will probably want to provide the explicit URL link, until 2.6 is released and the automatic reference can be used. The dev documentation URL will be `https://cantera.org/documentation/dev/sphinx/html/yaml/reactions.html#sec-yaml-blowers-masel`. Please also add a `TODO` in a comment here. Comments can be written in rST as:

```rST
.. TODO: Update the link once version 2.6 is released
```